### ISSUE TITLE
Staging Review finding: Links to PDFs in alerts open in same window, take users out of form flow

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#101257]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Veteran Facing Forms
**Product:** Direct Upload
**Feature:** Document Upload

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** Yes
**Collab Cycle Reviewer:** @erinrwhite (IA), @briandeconinck (Accessibility)

### Description

The links to the form PDF in the alerts open in the same browser window, which could cause users to lose progress on their forms.

This issue also occurs on the intro screen, which is okay since it isn't within the form flow yet. Opening that link in a new tab, or setting it to download rather than redirect the browser, could be helpful.

<details><summary>Link, screenshot or steps to recreate</summary>

![Image](https://github.com/user-attachments/assets/eccaa4b6-1328-41ac-9565-be685fd9f575)

</details>

## Recommended action

Open links to form PDFs in a new tab and add `(opens in a new tab)` using the `external` prop. It looks like `va-link` doesn't support both `external` and `download`, so it looks like we would need to remove `download` and lose the download icon --- but hopefully the DST could allow those two props to coexist?

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** [Link component](https://design.va.gov/components/link)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by tbaker1026** [comment]
Verified opens in a new window
